backend/src/services/encrypt/encrypt.service.ts

```diff
@@ -10,13 +11,23 @@ class Encrypt {
   constructor({ salt }: Constructor) {
     this.#salt = salt;
   }
+
   generateSalt(): Promise<string> {
     return genSalt(this.#salt);
   }
 
   encrypt(password: string, salt: string): Promise<string> {
     return hash(password, salt);
   }
+
+  public async compare(
+    data: string,
+    encryptionData: EncryptionData,
```

Comment:
```suggestion
    data: string,
    salt: string,
    passwordHash: string,
```
just provide three params